bodo/tests/test_series_errorchecking.py

```diff
@@ -405,8 +405,14 @@ def test_astype_non_constant_string(memory_leak_check):
     is not a compile time constant will produce a reasonable BodoError.
     """
 
+    @bodo.jit
+    def g(s):
+        # Add print to avoid constant inference to call the function at compile time
```

Comment:
It's better to add this comment to all print statements

Reply:
Done.